doc/specs/#1620 - Installer selector on multiple matches.md

```diff
@@ -0,0 +1,270 @@
+---
+author: Darshan Rander @SirusCodes
+created on: 2022-10-29
+last updated: 2022-11-11
+issue id: 1620
+---
+
+# Spec Title
+
+For [#1620](https://github.com/microsoft/winget-cli/issues/1620)
+
+## Abstract
+
+Currently, the winget.exe client simply lists all the matches and if the user wants to install the app they need to re-run the command with Id.
+
+The UX can be improved here by giving users an option to select the package instead of asking them to re-run the command with Id.
+
+This can be extended to `upgrade`, `show` and `uninstall` commands.
+
+## Inspiration
+
+Whenever I use Winget to install any application there are high chances that there are name collisions with other names and I get multiple suggestions in the table and it's annoying to copy the ID and rerun the command. I guess this is the case for a lot of others as well.
+
+## Solution Design
+
+If we find multiple packages then the user should get a selector.
+
+> It should not be shown if `--diable-interactivity` is set.
+
+### Settings
+
+The user needs to **opt-in** to this feature.
+
+Winget can use `--prompt` and `--no-prompt` flags to enable and disable it respectively.
+
+The user can change the default setting to always show the prompt by -
+
+```pwsh
+"packageDisambiguation": "prompt"
```

Comment:
```suggestion
```json
"interactivity" " {
  "packageDisambiguationPrompt": true
}
```
It would be best to use the existing settings structure

Reply:
This was something from @denelon's https://github.com/microsoft/winget-cli/pull/2650#issuecomment-1302541619

Reply:
It isn’t very clear what this is for then, as the way it is described here doesn’t match denelons comment.

I'll take a try at making some smaller tweaks and will create a PR into your branch

Reply:
Merged